scripts/build_doc_version_index.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+function silent_pushd() {
+  pushd $1 1>/dev/null 2>&1
+}
+
+function silent_popd() {
+  popd 1>/dev/null 2>&1
+}
+
+function all_version_directories_except_latest() {
+  find _docs -mindepth 1 -maxdepth 1 -not -path */_latest -type d
+}
+
+function all_versions() {
+  version_directories=$(all_version_directories_except_latest)
+  for version_directory in $version_directories; do
+    version=$(basename "$version_directory")
+    printf "$version\n"
+  done
+}
+
+function latest_version_files() {
+  silent_pushd _docs/_latest
+    find . -type f
+  silent_popd
+}
+
+function all_version_files() {
+  for dir in $(all_version_directories_except_latest); do
+  silent_pushd $dir
+    find . -type f
+  silent_popd
+  done 
+}
+
+function set_union_of_all_version_files() {
+  all_version_files | sort | uniq
+}
+
+function build_doc_version_index() {
+  
+  
+  > _data/doc_versions.yml
+  echo "# Auto-generated by ./scripts/build_doc_version_index.sh" >> _data/doc_versions.yml
+  echo "# DO NOT MODIFY" >> _data/doc_versions.yml
+
+  versions=$(all_versions | sort -rV)
+  for version in $versions; do
+    printf -- "- $version\n" >> _data/doc_versions.yml
+  done
+
+  > _data/doc_version_index.yml
+  echo "# Auto-generated by ./scripts/build_doc_version_index.sh" >> _data/doc_version_index.yml
+  echo "# DO NOT MODIFY" >> _data/doc_version_index.yml
+
+  # version_directories=$(all_version_directories_except_latest)
+  for file in $(set_union_of_all_version_files); do
+    filename="${file#./}"
+    printf "$filename:\n" >> _data/doc_version_index.yml
+
+    # for version_directory in $version_directories; do
```

Comment:
If this code isn't needed (commented out) go ahead and remove it please. Thanks.

Reply:
Removed.